odbcconfig/odbcinst.ini

```diff
@@ -4,4 +4,4 @@
 
 [EXAODBC]
 #Driver location will be appended in build environment:
-#DRIVER=<PATH TO DRIVER>
+DRIVER=/home/nic/Projects/sqlalchemy-exasol/driver/libexaodbc-uo2214lv1.so
```

Comment:
Local path. Should we provide a template file instead and add this file to gitignore

Reply:
Good point, did not happen the first time!